src/libs/antares/study/area/links.cpp

```diff
@@ -395,21 +395,56 @@ bool linkLoadTimeSeries_version_320_to_630(AreaLink& link, const AnyString& fold
     return ret;
 }
 
-/*
-            buffer.clear() << folder << SEP << link.with->id << ".txt";
-            link.setPathToDataFile(buffer);
-            ret = link.loadDataFromCSVfile(Matrix<>::optFixedSize | Matrix<>::optImmediate) && ret;
-*/
-bool linkLoadTimeSeries_version_630_to_820(AreaLink& link, const AnyString& folder)
+static bool linkLoadTimeSeries_version_630_to_820(AreaLink& link, const AnyString& folder)
```

Comment:
`linkLoadTimeSeries_version_630_to_810`, since format will change at 820, we want to exclude it. Also, add the "legacy" keyword to this function and the other 2 legacy functions.

Reply:
I think the "legacy" keyword is useless, as loading functions have a name explicit enough